CMakeLists.txt

```diff
@@ -97,7 +97,6 @@ if(RELEASE)
     set(BUILD_TYPE release)
 else()
     set(BUILD_TYPE dev)
-    string(APPEND CMAKE_CXX_FLAGS " -g")
```

Comment:
Removed because not needed? (just asking)

Reply:
It was passed twice.